cmd/cli/plugin/managementcluster/go.mod

```diff
@@ -0,0 +1,243 @@
+module github.com/vmware-tanzu/tanzu-framework/plugin/managementcluster
+
+go 1.17
+
+replace (
+	github.com/vmware-tanzu/tanzu-framework => ../../../../
```

Comment:
@Vandy-P , can we remove unused replace statements here as well as in the cluster/go.mod?

Reply:
Done